### DISCUSSION TITLE
How to limit user by category or something like this?

### DISCUSSION THREAD

**Discussion by tiicaa** [liedekef/events-made-easy#140] [Q&A]
I need to create 1 event with 3 different time options.
As a rule: Each duly logged in user will be able to choose only one time slot at a time within this event, not being able to choose another.
Another situation that would be necessary for me would be the event being created with different times for the same day, but when the user chooses a time, that specific time should be blocked for the others as a kind of limitation of vacancies, or even being able to launch separate events with limited spaces to solve this second situation, but the user would not be able to register for any other event of the same category. Any solution for both situations?

**Comment by liedekef** [comment]
Please do not open a discussion, an issue and send a personal mail asking the same question 3 times. See my answer in the issue posted.
